app/src/main/java/com/keylesspalace/tusky/components/conversation/ConversationEntity.kt

```diff
@@ -29,6 +29,7 @@ import com.keylesspalace.tusky.entity.TimelineAccount
 import com.keylesspalace.tusky.viewdata.StatusViewData
 import java.util.Date
 
+/** A reply-tree of statuses which have been saved to disk. */
```

Comment:
It's not a tree. The API for conversations returns the most recent (last) status in the thread. You can see this in Tusky if you open the DMs tab; each DM (conversation) is represented by the last status in the message thread. You have to tap on that message to see the full thread.

"saved to disk" is an implementation detail.

For commenting these I've found it helpful to link directly to the relevant Mastodon API documentation. For this one, it's a https://docs.joinmastodon.org/entities/Conversation/